### ISSUE TITLE
Passing chokidar options to run doesn't work

### ISSUE THREAD

**Issue by whitecolor** [fuse-box/fuse-box#611]
Passing chokidar options to run doesn't work.

**Comment by nchanged** [comment]
Could you be more specific please

**Comment by whitecolor** [comment]
Doesn't work:
```
const chokidar = { usePolling: true }

fuse.bundle('web.js')
  .instructions(`> web/web-fuse.ts`)
  .hmr()
  .watch() // <-- this calls `chokidar.watch`

fuse.run({ chokidar }) // <-- optioptions known only here
```

**Comment by whitecolor** [comment]
Also probably chokidar default `ignored` option should be `/node_modules/`. But at least docs should state that if one do not exclude modules folder it may significantly slow down things.

**Comment by nchanged** [comment]
Fixed in `2.2.0-beta.1`
